src/styles/main.scss

```
@@ -1,3 +1,4 @@
+@import "../styles/utilit/variable.scss";
```

Your main file is on the same level as the `utilit` folder, so you don't need this `../styles/`
Also, `.scss` is redundant there
```suggestion
@import "utilit/variable";
```